Pipfile

```diff
@@ -6,8 +6,6 @@ verify_ssl = true
 [dev-packages]
 
 [packages]
-bs4 = "*"
-lxml = "*"
```

Comment:
You'll still want to keep lxml in here since you are using it.

Reply:
I had deleted both files and they didn't regenerate properly, fixed with my latest commit after some informative pipenv reading